File: A2OJ/Ladder4_Codeforces_Div2A/1_Watermelon/watermelon.c
/*
 * Autor: Frederico Santos Gonçalves
 * Data: 12/01/2026
 *
 * Plataforma: A2OJ
 * Problema: Watermelon
 * Link: https://codeforces.com/problemset/problem/4/A
 * Dificuldade: 1
 */

#include <stdio.h>
#define SUCESSO (0)

int main() {
  int weight;
  scanf("%d", &weight);

  if(weight != 2 && weight%2 == 0) printf("YES");
  else printf("NO");

  return SUCESSO;
}

File: A2OJ/Ladder4_Codeforces_Div2A/001_Watermelon/watermelon.java
/*
 * Autor: Frederico Santos Gonçalves
 * Data: 12/01/2026
 *
 * Plataforma: A2OJ
 * Problema: Watermelon
 * Link: https://codeforces.com/problemset/problem/4/A
 * Dificuldade: 1
 */

import java.util.Scanner;

public class watermelon {
  static final Scanner scanner = new Scanner(System.in);

  public static void main(String[] args) {
      int weight = scanner.nextInt();
      if(weight != 2 && weight%2 == 0) System.out.print("YES");
      else System.out.print("NO");
  }
}
